.github/workflows/gradle-build-pr.yml

```diff
@@ -73,46 +73,46 @@ jobs:
       - name: "Checkout code"
         uses: actions/checkout@v2
       - name: "Whitelist Runner IP"
-        uses: broadinstitute/datarepo-actions@0.26.0
+        uses: broadinstitute/datarepo-actions@ms-integrationrework
         with:
           actions_subcommand: 'gcp_whitelist'
           role_id: ${{ secrets.ROLE_ID }}
           secret_id: ${{ secrets.SECRET_ID }}
       - name: "Check for an availble namespace to deploy API to and set state lock"
-        uses: broadinstitute/datarepo-actions@0.26.0
+        uses: broadinstitute/datarepo-actions@ms-integrationrework
         with:
           actions_subcommand: 'k8_checknamespace'
-          k8_namespaces: 'integration-1,integration-2,integration-3'
+          k8_namespaces: 'integration-6'
       - name: "Build docker container via Gradle"
-        uses: broadinstitute/datarepo-actions@0.26.0
+        uses: broadinstitute/datarepo-actions@ms-integrationrework
         with:
           actions_subcommand: 'gradlebuild'
       - name: "Deploy to cluster with Helm"
-        uses: broadinstitute/datarepo-actions@0.26.0
+        uses: broadinstitute/datarepo-actions@ms-integrationrework
         with:
           actions_subcommand: 'helmdeploy'
-          helm_secret_chart_version: '0.0.4'
-          helm_datarepo_api_chart_version: '0.0.19'
-          helm_datarepo_ui_chart_version: '0.0.9'
+          helm_create_secret_manager_secret_version: '0.0.6'
+          helm_datarepo_api_chart_version: '0.0.22'
+          helm_datarepo_ui_chart_version: '0.0.12'
```

Comment:
Is this just a bump to latest version, or are there other changes?

Reply:
- latest charts
- uniformed helm value names?
- uniformed deployments
- uniformed code for tests for api/ui